<?xml version="1.0" encoding="UTF-8"?>
<Examples>
  <Example>
    <Log>Elixir/h4cc@awesome-elixir/failed/504854205.log</Log>
    <Keywords>(throw), **, </Keywords>
    <Category>0</Category>
    <Chunk>** (throw) "Words not in order \"exos\" and \"ex_lttb\""
    check.exs:9: Awesome.Order.check_string_list_in_order/1
    check.exs:130: anonymous fn/2 in Awesome.test_file/1
    (elixir) lib/enum.ex:1755: Enum."-reduce/3-lists^foldl/2-0-"/3
    check.exs:129: Awesome.test_file/1
    (elixir) lib/code.ex:370: Code.require_file/2</Chunk>
  </Example>
  <Example>
    <Log>Elixir/h4cc@awesome-elixir/failed/506226859.log</Log>
    <Keywords>(throw), **, </Keywords>
    <Category>0</Category>
    <Chunk>** (throw) "Words not in order \"exjprop\" and \"ex_progress\""
    check.exs:9: Awesome.Order.check_string_list_in_order/1
    check.exs:130: anonymous fn/2 in Awesome.test_file/1
    (elixir) lib/enum.ex:1755: Enum."-reduce/3-lists^foldl/2-0-"/3
    check.exs:129: Awesome.test_file/1
    (elixir) lib/code.ex:370: Code.require_file/2</Chunk>
  </Example>
  <Example>
    <Log>Elixir/h4cc@awesome-elixir/failed/508448705.log</Log>
    <Keywords>(throw), **, </Keywords>
    <Category>0</Category>
    <Chunk>** (throw) "Words not in order \"thumbor_client\" and \"thumbnex\""
    check.exs:9: Awesome.Order.check_string_list_in_order/1
    check.exs:130: anonymous fn/2 in Awesome.test_file/1
    (elixir) lib/enum.ex:1755: Enum."-reduce/3-lists^foldl/2-0-"/3
    check.exs:129: Awesome.test_file/1
    (elixir) lib/code.ex:370: Code.require_file/2</Chunk>
  </Example>
  <Example>
    <Log>Elixir/h4cc@awesome-elixir/failed/513413329.log</Log>
    <Keywords>Error, **, </Keywords>
    <Category>1</Category>
    <Chunk>** (RuntimeError) Line does not match format: '[dummy](https://github.com/Vesuvium/dummy) - An Elixir mocking library that makes sense by exposing meck in the right way' Is there a dot at the end?
    check.exs:32: Awesome.parse_line/1
    check.exs:224: Awesome.validate_list_item/1
    check.exs:213: anonymous fn/2 in Awesome.check_list/1
    (elixir) lib/enum.ex:1755: Enum."-reduce/3-lists^foldl/2-0-"/3
    check.exs:213: Awesome.check_list/1
    check.exs:192: Awesome.iterate_content/1
    check.exs:117: Awesome.test_file/1</Chunk>
  </Example>
  <Example>
    <Log>Elixir/h4cc@awesome-elixir/failed/513586826.log</Log>
    <Keywords>Error, **, </Keywords>
    <Category>1</Category>
    <Chunk>** (RuntimeError) Line does not match format: '[medusa](https://github.com/Vesuvium/medusa) - A static site generator with support for Pug' Is there a dot at the end?
    check.exs:32: Awesome.parse_line/1
    check.exs:224: Awesome.validate_list_item/1
    check.exs:213: anonymous fn/2 in Awesome.check_list/1
    (elixir) lib/enum.ex:1755: Enum."-reduce/3-lists^foldl/2-0-"/3
    check.exs:213: Awesome.check_list/1
    check.exs:192: Awesome.iterate_content/1
    check.exs:117: Awesome.test_file/1</Chunk>
  </Example>
  <Example>
    <Log>Elixir/h4cc@awesome-elixir/failed/549739957.log</Log>
    <Keywords>(throw), **, </Keywords>
    <Category>0</Category>
    <Chunk>** (throw) "Words not in order \"yuri\" and \"Crawly\""
    check.exs:9: Awesome.Order.check_string_list_in_order/1
    check.exs:130: anonymous fn/2 in Awesome.test_file/1
    (elixir) lib/enum.ex:1755: Enum."-reduce/3-lists^foldl/2-0-"/3
    check.exs:129: Awesome.test_file/1
    (elixir) lib/code.ex:370: Code.require_file/2</Chunk>
  </Example>
  <Example>
    <Log>Elixir/h4cc@awesome-elixir/failed/557164268.log</Log>
    <Keywords>(throw), **, </Keywords>
    <Category>0</Category>
    <Chunk>** (throw) "Words not in order \"weather\" and \"coderplanets.com\""
    check.exs:9: Awesome.Order.check_string_list_in_order/1
    check.exs:130: anonymous fn/2 in Awesome.test_file/1
    (elixir) lib/enum.ex:1755: Enum."-reduce/3-lists^foldl/2-0-"/3
    check.exs:129: Awesome.test_file/1
    (elixir) lib/code.ex:370: Code.require_file/2</Chunk>
  </Example>
  <Example>
    <Log>Elixir/h4cc@awesome-elixir/failed/566095816.log</Log>
    <Keywords>(throw), **, </Keywords>
    <Category>0</Category>
    <Chunk>** (throw) "Duplicate links found"
    check.exs:68: Awesome.uniq_links/2
    (elixir) lib/code.ex:370: Code.require_file/2
    (mix) lib/mix/tasks/run.ex:82: Mix.Tasks.Run.run/1
    (mix) lib/mix/task.ex:294: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:58: Mix.CLI.run_task/2</Chunk>
  </Example>
  <Example>
    <Log>Elixir/h4cc@awesome-elixir/failed/569345345.log</Log>
    <Keywords>Error, **, </Keywords>
    <Category>1</Category>
    <Chunk>** (RuntimeError) Line does not match format: '[runhyve](https://runhyve.app) - The Virtual Machines Manager for Bhyve on FreeBSD' Is there a dot at the end?
    check.exs:32: Awesome.parse_line/1
    check.exs:224: Awesome.validate_list_item/1
    check.exs:213: anonymous fn/2 in Awesome.check_list/1
    (elixir) lib/enum.ex:1755: Enum."-reduce/3-lists^foldl/2-0-"/3
    check.exs:213: Awesome.check_list/1
    check.exs:192: Awesome.iterate_content/1
    check.exs:117: Awesome.test_file/1</Chunk>
  </Example>
  <Example>
    <Log>Elixir/h4cc@awesome-elixir/failed/569347444.log</Log>
    <Keywords>Error, **, </Keywords>
    <Category>1</Category>
    <Chunk>** (RuntimeError) Line does not match format: '[Runhyve](https://runhyve.app) - The virtual machines manager for bhyve on FreeBSD' Is there a dot at the end?
    check.exs:32: Awesome.parse_line/1
    check.exs:224: Awesome.validate_list_item/1
    check.exs:213: anonymous fn/2 in Awesome.check_list/1
    (elixir) lib/enum.ex:1755: Enum."-reduce/3-lists^foldl/2-0-"/3
    check.exs:213: Awesome.check_list/1
    check.exs:192: Awesome.iterate_content/1
    check.exs:117: Awesome.test_file/1</Chunk>
  </Example>
</Examples>
